TCMSApp/src/client/app/landing/landing.route.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('app.landing')
+        .run(appRun);
+
+    appRun.$inject = ['routerHelper'];
+    /* @ngInject */
+    function appRun(routerHelper) {
+        routerHelper.configureStates(getStates());
+    }
+
+    function getStates() {
+        return [
+            {
+                state: 'index',
+                config: {
+                    url: '/index',
+                    templateUrl: 'app/landing/landing.html',
+                    controller: 'LandingController',
+                    controllerAs: 'vm',
```

Comment:
maybe something like landingVm would be better. 
Why? So that we can access parent controller by its name from any child controller